master/internal/configpolicy/task_config_policy_intg_test.go

```diff
@@ -33,22 +70,52 @@ func TestPriorityAllowed(t *testing.T) {
 
 	// Priority is outside workspace limit.
 	smallerValueIsHigherPriority := true
-	ok, err = PriorityAllowed(w.ID, model.NTSCType, wkspLimit-1, smallerValueIsHigherPriority)
+	ok, err = PriorityUpdateAllowed(w.ID, model.NTSCType, wkspLimit-1, smallerValueIsHigherPriority)
 	require.NoError(t, err)
 	require.False(t, ok)
+	ok, err = PriorityUpdateAllowed(w.ID, model.ExperimentType, wkspLimit-1, smallerValueIsHigherPriority)
+	require.NoError(t, err)
+	require.True(t, ok)
 
 	globalLimit := 42
 	addConstraints(t, user, nil, fmt.Sprintf(`{"priority_limit": %d}`, globalLimit), model.NTSCType)
 
 	// Priority is within global limit.
-	ok, err = PriorityAllowed(w.ID, model.NTSCType, wkspLimit-1, true)
+	ok, err = PriorityUpdateAllowed(w.ID, model.NTSCType, wkspLimit-1, true)
+	require.NoError(t, err)
+	require.True(t, ok)
+	ok, err = PriorityUpdateAllowed(w.ID, model.ExperimentType, wkspLimit-1, true)
 	require.NoError(t, err)
 	require.True(t, ok)
 
 	// Priority is outside global limit.
-	ok, err = PriorityAllowed(w.ID+1, model.NTSCType, globalLimit-1, true)
+	ok, err = PriorityUpdateAllowed(w.ID+1, model.NTSCType, globalLimit-1, true)
 	require.NoError(t, err)
 	require.False(t, ok)
+	ok, err = PriorityUpdateAllowed(w.ID+1, model.ExperimentType, globalLimit-1, true)
```

Comment:
```suggestion
         // Global Experiment constraints have not been set
	ok, err = PriorityUpdateAllowed(w.ID+1, model.ExperimentType, globalLimit-1, true)
```

Can we make this change to clarify that the function behavior isn't different for experiments, but this is just a check due to test setup (which I love that you're checking this!)?

Reply:
Oh funny story! I've started looking at codecov to see which lines aren't covered by tests. A lot of them aren't useful things to test (like checking for errors that shouldn't be occurring anyways). But sometimes I find a bit of logic that really should be covered by a simple extra case like this.